src/include/os_fs.i

```diff
@@ -114,7 +114,8 @@ __wt_fs_remove(WT_SESSION_IMPL *session, const char *name, bool durable)
 
 	file_system = S2C(session)->file_system;
 	wt_session = (WT_SESSION *)session;
-	ret = file_system->fs_remove(file_system, wt_session, path, durable);
+	ret = file_system->fs_remove(
+	    file_system, wt_session, path, durable ? WT_FS_DURABLE : 0);
```

Comment:
I would suggest exposing the flags all the way up to the callers of `wt_fs_remove/rename` rather than keep the boolean as an arg.  If we need lower level flags in the future, we'll need more args or convert to flags for these functions anyway.